.env

```diff
@@ -0,0 +1,5 @@
+POSTGRES_PORT=
+POSTGRES_HOST=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+POSTGRES_DB=
```

Comment:
the .env file is still in the .gitignore so shouldn't come along with further pull requests, but the placeholder variables are here without value so people can fill them in locally

Reply:
Interesting trick! I did not know about it but it seems handy.

Reply:
Looking at it now this might not seem to work as I thought, I'll check it later and if so I'll rename it to a env.example file

Reply:
Moved to example.env as the .gitignore indeed did not ignore this file

Reply:
I think you have to remove the file and commit the change for .gitignore to actually ignore it

Reply:
There already is a rule for the .env file in the current gitignore 😃